src/controllers/transaction/TransactionMaterialController.ts

```diff
@@ -56,14 +60,16 @@ export default class TransactionMaterialController extends AbstractController<
 	 * @param res Express Response
 	 */
 	private getTxArtifacts: RequestHandler = async (
-		_req,
+		{ query: { noMeta } },
 		res
 	): Promise<void> => {
 		const hash = await this.api.rpc.chain.getFinalizedHead();
 
+		const noMetaArg = noMeta === 'true' ? noMeta : undefined;
```

Comment:
Is there not some way to convert this to a proper boolean using the Express middleware or something?

Reply:
Converted to a bool, right here. I think changing the query param to be a boolean in the middleware might be to much given we just need it in this service and its a one liner